### ISSUE TITLE
OA Project Intake

### ISSUE THREAD

**Issue by JMensch** [open-austin/project-ideas#138]
### What problem are you trying to solve?

Answer here.

### Who will benefit (directly and indirectly) from your project?

Answer here.

### What other resources/tools are currently serving the same need? How does your project set itself apart?

Answer here.

### Where can we find any research/data available/articles?

Links here.

### What help do you need now?

Answer here.

### What are the next steps (validation, research, coding, design)?

Answer here.

### How can we contact you outside of Github(list social media or places you're present)?

Answer here.

----

#### Checklist for NEW ideas :baby:
Hey, you're official! You're now part of the growing civic hacking community in Austin. Here's a few things to get started (a couple you've probably already done).

- [x] Create this idea issue
- [x] Flesh out the who, where, and what questions above
- [ ] Start the conversation about this idea on [Slack](https://openaustin-slackin.herokuapp.com/) Replace this link to the [#general](https://open-austin.slack.com/messages/C036FLME2) channel with your project's preferred channel.

#### Checklist for ACTIVE projects :fire:
Let's get this project started! When this idea starts taking off, the Projects Core Team will start helping this project's lead(s) out with project management and connecting you to resources you may need. To get there, please complete and check off the following:

- [x] Post an update at least once a month to this issue. Use [BASEDEF](https://github.com/open-austin/cheatsheet-basedef) for ideas, but it's ok even if your update is just "nothing new happened this month" or "we saw a small increase in traffic to our app this month". If there's no activity for two months, that's no problem, life happens. We'll just label this as `backlog` so others know you'll get back to it when you have the time. If nobody hears from you at all in more than two months, we may mark it as `abandoned` so that others can pick up this idea and run with it.
- [ ] Take 30 minutes to complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101) 
- [ ] Create a [GitHub repository](https://github.com/new). Ask for help setting up permissions if you want your repository to be within the [Open Austin Github organization](https://github.com/open-austin).
- [ ] Create a [README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [ ] [Create issues](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#assignment--add-your-first-issue) to describe each task that you plan to do or need help with and how a contributor can get started on that task. You might start and stop a lot, so consider issues as your to-do list.
- [ ] Create a [team](https://help.github.com/articles/creating-a-team/) for your core contributors
  - This will make it easier for you to manage your github repo access. People on a team have the same level of access. Admin access will allow your trusted contributors to make changes as needed. 
  - You can remove and add people to your team as needed. 
  - View some of our teams: [https://github.com/orgs/open-austin/teams](https://github.com/orgs/open-austin/teams)
  - Note: You can also allow collaborators outside of your team and give them more limited access.
- [ ]   Create a [user group](https://get.slack.help/hc/en-us/articles/212906697-User-Groups) in Slack so you can "@" your core contributors all at once, without bothering other people who use the Slack channel. You'll need permission from a Slack admin, so just mention @leadership on Slack to get this set up.
- [ ]   Create a [Google Drive](https://drive.google.com), [Dropbox](https://www.dropbox.com/), or other cloud storage to share larger files. Github and Data.World are good for code and data, respectively, especially when you need version control. But they're not good for very large files, documentation, articles, etc. A cloud storage option will allow you to easily share, create, and collaborate on documents with your team and help organize ideas and thoughts.
  - Doing this early on can help your team stay organized and to onboard new contributors who wouldn't have access to files you all have shared over email.

#### Checklist for FEATURED Projects :tada:
To have your project FEATURED on [Open-Austin.org](https://open-austin.org/), complete the following documentation. In past projects, well-documented featured projects have more contributions than other projects.

- [ ] In your README, link to the [Open Austin Code of Conduct](https://www.open-austin.org/about/#code-of-conduct) or write your own code of conduct.
- [ ] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
- [ ] [Create file: CONTRIBUTING.md](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md) so others know [how they can contribute](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/).
- [ ] Create an easily shareable project management artifact, like a [Civic Tech Canvas](https://cityofaustin.github.io/civic-tech-canvas/) or [Open Canvas](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/)
- [ ] [Create an issue](https://github.com/open-austin/open-austin.github.io/issues/new) on the open-austin.github.io repo with the title `Add [my project] to projects page`. An Open Austin leader will review this issue and post your project :balloon:
- [ ] [Tell the City of Austin](http://austintexas.gov/page/mobile-app-library). If your idea is in a shareable format and can benefit people around the city, go to that site and follow the instructions on the bottom of the page to showcase your work there.

If you get stuck at any point, feel free to reach out to the leadership team on Slack by adding @leadership to your message. We're here to help you make real changes to our city.

**Comment by JMensch** [comment]
We still have some discovery to do, I think a good next step is to solicit feedback from members via a form or slack channel (or both). Thoughts?

**Comment by olitreadwell** [comment]
Perhaps a somewhat structured process? 
- Invite folks to comment on this issue. 
- Create a beta version of a form to solicit feedback about the project creation process
- Schedule a Zoom meeting or a breakaway session in an upcoming Open Ausitin meeting where interested folks can contribute feedback in a more open-ended manner

**Comment by olitreadwell** [comment]
My hopes for a project intake would be as much educational as it was inviting. 

Teach about design thinking and to some varying degree validation, and minimal viable product, while also enabling folks to think deeply about an idea.

**Comment by olitreadwell** [comment]
I for one believe that project intake occurring on Github or any unfamiliar site where you have to create a new account and learn the UI will prevent new projects from landing, whether thought out or not.

**Comment by JMensch** [comment]
> Perhaps a somewhat structured process?

Definitely! I like what you have. We need to take an initial stance on the groundwork before inviting feedback on a solution, though. At minimum:

- What constitutes a project?
  - What is the minimum required for something to be considered a project?
  - What kind of project categories (github tags) do we want? Which of these are required entry? 
  - Do we differentiate between ideas and projects?

> My hopes for a project intake would be as much educational as it was inviting.
> Teach about design thinking and to some varying degree validation, and minimal viable product, while also enabling folks to think deeply about an idea.

<3<3<3 this. totally agree.

> I for one believe that project intake occurring on Github or any unfamiliar site where you have to create a new account and learn the UI will prevent new projects from landing, whether thought out or not.

Also agree. I figure we can build/use some sort of tech that feeds into github (e.g. your form idea)

@mscarey any thoughts before I move this conversation to the slack project channel?

**Comment by mscarey** [comment]
For me, the most important feature of a form is the ability to see all the questions before I start writing. So that's one feature I'd like to preserve if the form gets redesigned.

If the intake process gets any more difficult than it already is, and especially if it requires a design thinking phase, then I think it's really important for people to be able to save and share their project ideas even when the intake process is incomplete.

And I wouldn't want to require people to tag their project ideas with the technologies their project will use too early in the process, because project champions often don't know what the right technology choices are. But people probably will want the option to add technology tags. A common complaint among project leaders is that they waste time with prospective volunteers who want to use a language or framework that's not going to work with the project's existing code.

**Comment by mscarey** [comment]
I closed a bunch of project ideas that, for one reason or another, weren't currently suitable to have people start working on them. Maybe that'll make the project onboarding process easier for new people.

**Comment by olitreadwell** [comment]
> For me, the most important feature of a form is the ability to see all the questions before I start writing. So that's one feature I'd like to preserve if the form gets redesigned.

I understand this desire, though might argue that the current form asks a bunch of questions that are then denoted as being unnecessary on initial submission. My thought is that there is a staged submission process that saves the information while educating folks hoping to submit ideas without overwhelming them.

> If the intake process gets any more difficult than it already is, and especially if it requires a design thinking phase, then I think it's really important for people to be able to save and share their project ideas even when the intake process is incomplete.

> And I wouldn't want to require people to tag their project ideas with the technologies their project will use too early in the process, because project champions often don't know what the right technology choices are.

I'm in full agreement with both of these points @mscarey 

---

One recommendation I really appreciated from @mscarey and @mateoclarke was that before creating a new project intake process it'd be valuable to do some user research. 

Our suggestion was that during an upcoming session, find someone without a github user account and give them a prompt of "you have an idea for a civic tech project that you are determined to get started on, you've heard about 'open austin' vaguely show us how you would go ahead and work to get the opportunity to pitch your idea. "
